addons/popochiu/engine/others/popochiu_utils.gd

```diff
@@ -80,4 +80,18 @@ static func get_click_or_touch_index(event: InputEvent) -> int:
 	return index
 
 
+## For each element in [param array] calls [param callback] passing the element as a parameter. If
+## any of the calls returns [code]true[/code], then this function returns [code]true[/code],
+## otherwise [code]false[/code] is returned.[br][br]
+## This is an alternate version for [method Array.any] which doesn't stops the execution if one
+## of the results is [code]true[/code] in a large array.
+static func any(array: Array, callback: Callable) -> bool:
```

Comment:
Now that I think about it, I may have misunderstood how `Array.any` works. It might stop execution as soon as **one element** returns `true` (that would explain a lot of things... My stupidity being one of them).

In that case, the purpose of this function is to execute `callback` for each element of the array and return `true` if at least one of those calls returned `true`, but without stopping execution because of that. I'm between `any_exhaustive` and `any_non_short_circuir`.